packages/rust-base/Dockerfile

```
@@ -0,0 +1,28 @@
+# Use the official Rust image as the base
+FROM rust:1.92
+ 
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)